mls-rs/src/group/commit/processor.rs

```diff
@@ -282,6 +325,16 @@ impl<C: ClientConfig> CommitProcessor<'_, C> {
     pub fn authenticated_data(&self) -> &[u8] {
         &self.0.authenticated_data
     }
+
+    pub fn required_psks(&self) -> Vec<ExternalPskId> {
+        self.0
+            .proposals
+            .psk_proposals()
+            .iter()
+            .filter_map(|p| p.proposal.external_psk_id())
+            .cloned()
+            .collect()
+    }
```

Comment:
What about resumption PSKs? They're handled explicitly in other places which I think makes sense

Reply:
fixed